### ISSUE TITLE
Enhanced Item Sheet

### ISSUE THREAD

**Issue by Muwak77** [n3dst4/gumshoe-fvtt#646]
I'd like to suggest an addition to an item sheet. Based on the permission-system we used for NPC id would me a whopper if we could set a minimum permission for the dynamig Item fields.

So a GM could decide which Information is secret to players. 

Alternatively a simple GM-Notes field could be added, which would also do the trick.

**Comment by n3dst4** [comment]
This is a very neat idea. Shouldn't be too hard to do. The only challenge is shoehorning the extra data into the settings UI 😄

**Comment by Muwak77** [comment]
If you don't mind, I can try my hand at it. It would just be cool if you could give me a tip on how to put in a pull request, in my professional projects I can simply make a branch, but I haven't figured out how to do that in an external Github project yet.

**Comment by n3dst4** [comment]
Definitely! I can see you already have a fork, shich is step 1.

Step 2 is to get your main branch updated to mine.

Then write your code and push to a new branch in your fork.

Once your happy, if you go yuour fork on GitHub it will be promting you to open a PR.

Once your on Discord (link on the other ticket) I can walk you through the process if you like.

**Comment by Muwak77** [comment]
ah okay so i can sand a PR from MY repo to yours

**Comment by n3dst4** [comment]
Yep, exactly! That's the model for open-source collaboration on GitHub - you have your own fork so you can make as many branches as you like without distracting anyone, then you can offer those back to the parent project as a PR
